perllib/FixMyStreet/SendReport/Email.pm

```diff
@@ -63,14 +63,29 @@ sub get_template {
 
 sub send_from {
     my ( $self, $row ) = @_;
+
+    my $cobrand = $row->get_cobrand_logged;
```

Comment:
I think perhaps an easier way of doing this would be to have a SendReport::Email subclass (there are already a few), perhaps SendReport::Email::Bexley, that would use the do_not_reply/council_name without having to do any checking; you could then set the send method to for this category in the admin.